src/plugins/condition/components/inspector/StylesView.vue

```diff
@@ -421,11 +421,11 @@ export default {
         },
         addConditionSet() {
             let conditionSetDomainObject;
-            const handleItemSelection = (item) => {
+            function handleItemSelection(item) {
```

Comment:
Again, there seem to be weird inconsistencies here around arrow functions.

Reply:
Seems to be because the second function had internal refs to `this`. When I used `let self = this` and changed to self, it was also flagged.

Reply:
Looks like you're right, the ESLint source seems to have handling for arrow functions that contain `this`. Despite it not being documented.

`dismissDialog` is complex enough that it should be a member function of StylesView but I'm going to let it slide for now.